docusaurus/docs/iOS/03-guides/15-migration-from-dolby.mdx

```diff
@@ -0,0 +1,826 @@
+
+2. **Livestream Tutorial**:
+   - **Link**: [Stream Livestream Tutorial](https://getstream.io/video/sdk/ios/tutorial/livestreaming/)
+   - **Overview**: This resource guides you through creating a livestreaming application. It includes details on setting up live video streams and managing interactive features.
+   - **Useful For**: Those interested in integrating live video streaming functionalities into their applications.
+
+3. **Video Calling Tutorial**:
+   - **Link**: [Stream Video Calling Tutorial](https://getstream.io/video/sdk/ios/tutorial/video-calling/)
+   - **Overview**: This tutorial provides insights into developing a video calling feature using Stream SDK. It details the implementation of video calls, handling call states, and UI integration.
+   - **Useful For**: Developers aiming to add or enhance video calling capabilities in their apps.
+
+Leveraging these tutorials will not only assist you in the migration process but also expand your understanding and skills in building diverse applications with the Stream SDK. Each tutorial is designed to address specific use cases and functionalities, making them valuable resources for developers at any skill level. Additionally, for any specific queries or support, the Stream developer community and their support team are available to assist you.
+
+
+## Conclusion
+
+As we reach the end of this migration guide, it's clear that the Stream SDK stands out as a more feature-rich and user-friendly option compared to Dolby.io SDK, especially for developers working in the modern iOS ecosystem. Stream's commitment to leveraging the latest technologies in Swift and SwiftUI makes it not only a future-proof choice but also ensures a smoother and more efficient development experience.
+
+The Stream SDK's extensive feature set, including its built-in permission system, advanced call management capabilities, and optimized support for SwiftUI, offers a comprehensive toolkit for building sophisticated Voice and Video applications. These features not only enhance the functionality of your app but also provide a more engaging user experience.
+
+Furthermore, Stream's utilization of modern Swift technologies, like concurrency, makes it a more appealing option for developers looking to harness the full potential of the latest iOS development trends. This modern approach simplifies the coding process, reduces development time, and improves overall app performance.
+
+In summary, by migrating to the Stream SDK, you're not just updating your tech stack; you're taking a significant step forward in delivering a superior audio conferencing experience to your users. Stream SDK empowers you to build more dynamic, scalable, and feature-rich applications with ease and efficiency. Embrace this migration as an opportunity to elevate your AudioRoom app to new heights, leveraging the best of what modern technology has to offer.
```

Comment:
This sounds like you have to update your tech stack to use us, which is not correct. Maybe instead we can mention that our modern stack works with older tech as well.